sys/arduino/include/arduino.hpp

```diff
@@ -34,7 +34,7 @@ extern "C" {
 enum {
     INPUT,              /**< configure pin as input */
     OUTPUT,             /**< configure pin as output */
-    INPUT_PULLUP,       /**< configure pin as input with pull-up resistor */
+    INPUT_PULLUP        /**< configure pin as input with pull-up resistor */
```

Comment:
I'm not sure I like this (in general). If we (hypothetically, since this specific struct is a bad example for this, but say for example `gnrc_nettype_t`) want to add to this enum it would edit two lines rather than one.